modules/Microsoft.DataFactory/factories/.test/parameters.json

```diff
@@ -61,6 +61,23 @@
             "value": {
                 "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-<<namePrefix>>-az-msi-x-001": {}
             }
+        },
+        "privateEndpoints": {
+            "value": [
+                {
+                    "subnetResourceId": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Network/virtualNetworks/adp-<<namePrefix>>-az-vnet-x-001/subnets/<<namePrefix>>-az-subnet-x-005-privateEndpoints",
+                    "service": "dataFactory"
+                }
+            ]
+        },
+        "cMKUserAssignedIdentityResourceId": {
```

Comment:
Same as other PRs comment. Should we test CMK through a dedicated test `encr.parameters.json`?